src/hooks/useAccess.ts

```diff
@@ -1,25 +1,56 @@
 import { RequestBundle, apiRequest } from './apiRequest';
+import { FlashContext } from '../contexts/FlashContext';
+import { useContext } from 'react';
 
-const ACCESS_BASE_URL = process.env.NEXT_PUBLIC_ACCESS_BASE_URL;
 
```

Comment:
Let's keep it the original instead of directly interpolating the env var. There should be a check ensuring that the env var exists 

```suggestion
const ACCESS_BASE_URL = process.env.NEXT_PUBLIC_ACCESS_BASE_URL;

```